hive/server/condenser_api/cursor.py

```diff
@@ -349,7 +349,8 @@ def last_month():
        SELECT id FROM hive_posts
         WHERE parent_id IN (SELECT id FROM hive_posts
                              WHERE author = :parent
-                               AND is_deleted = '0') %s
+                               AND is_deleted = '0'
+                          ORDER BY id DESC) %s
```

Comment:
If you have an opportunity to `EXPLAIN` this query to make sure the indices are getting hit, that might be valuable :¬)